src/uwtools/cli.py

```diff
@@ -351,6 +354,60 @@ def _dispatch_file_link(args: Args) -> bool:
     )
 
 
+# Mode invoke
+
+
+def _add_subparser_invoke(subparsers: Subparsers) -> ModeChecks:
+    """
+    Subparser for mode: invoke
+
+    :param subparsers: Parent parser's subparsers, to add this subparser to.
+    """
+    parser = _add_subparser(subparsers, STR.invoke, "Invoke external driver.")
+    required = parser.add_argument_group(TITLE_REQ_ARG)
+    _add_arg_module(required)
+    _add_arg_classname(required)
+    _add_arg_task(required)
+    _add_arg_schema_file(required)
+    optional = _basic_setup(parser)
+    _add_arg_module_dir(optional)
+    _add_arg_config_file(optional)
+    _add_arg_cycle(optional)
+    _add_arg_leadtime(optional)
+    _add_arg_batch(optional)
+    _add_arg_dry_run(optional)
+    _add_arg_graph_file(optional)
+    _add_arg_key_path(
+        optional,
+        helpmsg="Dot-separated path of keys leading through the config "
+        "to the driver's configuration block",
+    )
+    return {STR.invoke: _add_args_verbosity(optional)}
```

Comment:
Users won't be able to add additional args, huh?

Reply:
I think external drivers should conform to UW conventions, which define a set of supported arguments. Beyond that,  I think configuration should go in the YAML config. (I have ongoing misgivings about even `cycle`, `leadtime`, and `batch` due to the complications they've introduced. I know we want to support using the same base config for multiple cycles/leadtimes, but -- and this may not have always been the case -- that can be done, simply if a bit more verbosely, by pre-processing a driver config with `config realize`. It can even be done with pipes, so no intermediate files are necessary. That said, the flags _are_ handy, and if the CLI doesn't get further complicated, maybe we've dealt well enough with the complexity.)

Reply:
Just making sure I was correct in this assumption. Thanks for expanding on the excellent reasons not to go beyond those flags we officially support.